tests/utils/io_test.py

```diff
@@ -47,24 +47,24 @@ def mock_red_hat(path):
 
 class IoTests(TestCase):
     def test_normalize_path(self):
-        self.assertEqual("/already/a/normalized/path", io.normalize_path("/already/a/normalized/path"))
-        self.assertEqual("/not/normalized", io.normalize_path("/not/normalized/path/../"))
-        self.assertEqual(os.path.expanduser("~"), io.normalize_path("~/Documents/.."))
+        assert io.normalize_path("/already/a/normalized/path") == "/already/a/normalized/path"
+        assert io.normalize_path("/not/normalized/path/../") == "/not/normalized"
+        assert os.path.expanduser("~") == io.normalize_path("~/Documents/..")
```

Comment:
Should we reverse the order here so the call to `io.normalize_path` comes first? Then this is consistent with the other assertions.

Reply:
Nice catch, thanks